### ISSUE TITLE
 Russian translation

### ISSUE THREAD

**Issue by Messia91** [Treeofsavior/EnglishTranslation#1463]
Hello, may I make a Russian translation also on an international server?

**Comment by Sunaries** [comment]
You are free to create a Russian translation, but I do not know if @imcgames supports an official repository at this moment. I do not know whether other people already begun working on a different website.

**Comment by Messia91** [comment]
Yes there are people that make the translation, and I chisla.Perevod of them already and ready for about 60% of an item is a quest and a bit of skill.
